src/content/docs/mobile-apps/quick-actions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Quick actions 
+tags:
+  - Mobile apps
+  - New Relic mobile apps
+metaDescription: Use quick actions to display your four favorite entities. 
+---
+
+
+import mobileAppsQuickActionAndroid from 'images/mobile-apps_screenshot_full_android_quick_action.webp'
+
+import mobileAppsQuickActioniOS from 'images/mobile-apps_screenshot_full_ios_quick_action.webp'
+
+
+To keep up in the fast-paced world of engineering and observability, use quick actions to get speedy access to common features of New Relic&mdash;right on the home screen. 
+
+
+Quick actions displays up to four of your favorite entities. If you have more than four favorites, you’ll see the most recently used ones. If you haven't set up any favorites, the display shows links to the most-used capabilities (APM, alerts, synthetics, dashboards). 
```

Comment:
How do you set up your favorites? Is it different for mobile?

Reply:
@homelessbirds this is all for mobile apps, and I'm thinking that setting up favorites is a common task on mobile apps.